runtimes/node-18.0/build.sh

```diff
@@ -8,13 +8,21 @@ cp -R /usr/code/* /usr/builds
 
 # Install User Function Dependencies if package.json exists
 cd /usr/builds
-if [ -f package.json ]; then
-  npm install
+
+# Ensure package.json exists
+if [[ ! -f "package.json" ]]; then
+    mv /usr/local/src/package.json.fallback /usr/builds/package.json
 fi
 
-mkdir -p node_modules
+INSTALL_COMMAND=${1:-'npm install'}
+BUILD_COMMAND=${2:-''}    
+
+eval "$INSTALL_COMMAND"
```

Comment:
I think we're missing the goal here. I think Open Runtimes should only accept and entrypoint. In the entrypoint we can pass a chain of commands for either build or install from an external platform (e.g. Appwrite). This will allow runtimes to behave exactly the same for both build and execution.

In that external platform (like Appwrite) we can perform validation and ensure that the final command for execution is starting an HTTP server or in the build that the first command is installation or whatever we need.

This PR should actually remove a lot of code rather than adding.